### DISCUSSION TITLE
Canadian Channels N/A?

### DISCUSSION THREAD

**Discussion by baazin16** [iptv-org/community#460] [Q&A]
Hello, I'm wondering if anyone else is experiencing issues with Canadian Channels (E.g CTV News, BNN - Canada's News Network). Whenever I try to stream it says "There are no live events at this time please check back later" but I know there should be live events ongoing.

Any insights appreciated, thanks.

**Comment by HNJAMeindersma** [comment]
Is this a technical HTTP error message you're talking about. Or are you getting a valid video stream where the image on-screen says the error you're talking about?

I'm aware that more and more linear content providers / broadcasters have multiple versions of "a single channel". They use these to, for example, provide different localized commercials to a specific region. But this technique can also be used to block certain content from being viewed outside a rightsholders region. This happens allot with IPTV streams. For example: NHK World-Japan temporarily overwrites their online stream when they air baseball footage, which they are only allowed to show domestically inside Japan. You can also see a placeholder image on the IPTV stream from CNN when they air commercials, where they overwrite that placeholder image with different commercial-blocks depending on the region that the content-signal is provided to. I expect that this might be going on with your Canadian live event streams. If you're watching outside the borders of Canada, using a VPN with Canadian IP address might solve your issue. But the issue might also be the fact that it's still IPTV on which some content may not be aired due to legal rights.
